src/stories/modal-pause/modal-pause.scss

```diff
@@ -0,0 +1,75 @@
+$MODAL_PAUSE_CONTAINER_WIDTH: 550px;
+
+.modal-pause {
+  flex-direction: column;
+  align-items: center;
+}
+
+.modal-pause__container {
+  margin: 0 24px;
+  padding-top: 96px;
+  max-width: $MODAL_PAUSE_CONTAINER_WIDTH;
+  width: 100%;
+
+  @include breakpoint-s {
+    margin: 0 45px;
+  }
+}
+
+.modal-pause__button {
+  display: grid;
+  justify-content: center;
+}
+
+.modal_pause__text-link {
+  .link-tag {
+    font-size: 14px;
+    padding-left: 5px;
+  }
+}
+
+.datepickers {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 80px;
+
+  input {
+    border: 1px solid #dbdbdb;
+    box-sizing: border-box;
+    background-color: #f6f5f0;
+    min-width: 200px;
+    height: 50px;
+    padding: 0 20px;
+    color: #484848;
+    @extend %text-button-placeholder;
+
+    @include breakpoint-s {
+      min-width: 250px;
+    }
+  }
+
+  .datepicker {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  [type="date"] {
+    background: #f6f5f0 url("https://cdn1.iconfinder.com/data/icons/cc_mono_icon_set/blacks/16x16/calendar_2.png") 90% 50% no-repeat ;
```

Comment:
Do we have a variable for this color somewhere?

Reply:
Good catch, there was a variable that I could use 👏